src/main/java/appointment/watcher/Watcher.java

```diff
@@ -1,14 +1,23 @@
 package appointment.watcher;
 
+import appointment.watcher.exception.CustomException;
+import appointment.watcher.exception.ExceptionType;
 import appointment.watcher.service.WatcherService;
 import com.twilio.Twilio;
 import com.twilio.rest.api.v2010.account.Message;
 import com.twilio.type.PhoneNumber;
+import lombok.extern.slf4j.Slf4j;
 
+@Slf4j
 public class Watcher {
 
+    private static final String TWILIO_ACCOUNT_SID = "<twilio account sid>";
+    private static final String TWILIO_AUTH_TOKEN = "<twilio auth token>";
+    private static final String FROM_PHONE_NUMBER = "<from phone number>"; // Select from Twilio
+    private static final String TO_PHONE_NUMBER = "<to phone number>";
```

Comment:
May be we can move this into config file?

Reply:
Not using Spring here. Any easy way to read a config file in core java? I don't feel like writing a parser for a .config file lol

Reply:
Ah, I meant we can just move to constants file for now! Yes, we probably can use properties file but that's okay for now!